src/amber/cli/commands/database.cr

```diff
@@ -52,17 +52,19 @@ module Amber::CLI
             `crystal db/seeds.cr`
             Micrate.logger.info "Seeded database"
           when "migrate"
-            Micrate::Cli.run_up
+            # TODO(kevinsjoberg): Remove begin/rescue as Micrate properly
```

Comment:
@KevinSjoberg I like your ownership but I'd rather not start dropping TODO comments around the code. If Micrate fixes this bug, the code can be removed later but it's not likely to break anything if that happens and it lives on.

Reply:
@robacarp makes sense. I'll remove that.